indexer/sql-migration/src/lib.rs

```diff
@@ -1,8 +1,8 @@
-pub use sea_orm_migration::prelude::*;
 mod idens;
-
 mod m20220101_000001_create_table;
 
+pub use sea_orm_migration::prelude::*;
+
 pub struct Migrator;
```

Comment:
This is generated automatically when creating migration, this will be rewritten all the time.